front/tests/pages/stdcm-page-model.ts

```diff
@@ -307,68 +320,134 @@ class STDCMPage {
-    await expect(this.originArrival).toHaveValue('preciseTime');
-    await expect(this.dateOriginArrival).toHaveValue('17/10/24');
-    await expect(this.timeOriginArrival).toHaveValue('00:00');
-    await expect(this.toleranceOriginArrival).toHaveValue('-30/+30');
-    await this.dynamicOriginCh.selectOption('BC');
-    await this.originArrival.selectOption('respectDestinationSchedule');
+    expect(originCiValue).toContain(ORIGIN_DETAILS.suggestionResult);
+    // Verify default values
+    await expect(this.dynamicOriginCh).toHaveValue(ORIGIN_DETAILS.defaultChValue);
+    await expect(this.originArrival).toHaveValue(ORIGIN_DETAILS.defaultArrivalType);
+    await expect(this.dateOriginArrival).toHaveValue(ORIGIN_DETAILS.defaultArrivalDate);
+    await expect(this.timeOriginArrival).toHaveValue(ORIGIN_DETAILS.defaultArrivalTime);
+    await expect(this.toleranceOriginArrival).toHaveValue(ORIGIN_DETAILS.defaultTolerance);
+    // Update and verify origin values
+    await this.dynamicOriginCh.selectOption(ORIGIN_DETAILS.updatedChValue);
+    await expect(this.dynamicOriginCh).toHaveValue(ORIGIN_DETAILS.updatedChValue);
+    await this.originArrival.selectOption(ORIGIN_DETAILS.updatedArrivalType);
+    await expect(this.originArrival).toHaveValue(ORIGIN_DETAILS.updatedArrivalType);
+    // Verify fields are hidden
     await expect(this.dateOriginArrival).not.toBeVisible();
     await expect(this.timeOriginArrival).not.toBeVisible();
     await expect(this.toleranceOriginArrival).not.toBeVisible();
   }
 
   // Fills and verifies destination details based on selected language
   async fillAndVerifyDestinationDetails(selectedLanguage: string) {
+    const DESTINATION_DETAILS = {
+      destinationInput: 'South',
+      suggestionResult: 'South_station',
+      defaultChValue: 'BV',
+      defaultArrivalType: 'asSoonAsPossible',
+      updatedArrivalType: 'preciseTime',
+      defaultArrivalDate: '17/10/24',
+      defaultArrivalTime: '00:00',
+      defaultTolerance: '-30/+30',
+      updatedDate: '18/10/24',
+      updatedHour: '01',
+      updatedMinute: '35',
+      updatedTimeValue: '01:37',
+      toleranceNegative: '-75',
+      tolerancePositive: '+45',
+    };
     const translations = selectedLanguage === 'English' ? enTranslations : frTranslations;
-    await this.dynamicDestinationCi.fill('South');
+    // Fill Destination Input and Verify Suggestions
+    await this.dynamicDestinationCi.fill(DESTINATION_DETAILS.destinationInput);
     await this.verifyDestinationSouthSuggestions();
     await this.suggestionSS.click();
     const destinationCiValue = await this.dynamicDestinationCi.getAttribute('value');
-    expect(destinationCiValue).toContain('South_station');
-    await expect(this.dynamicDestinationCh).toHaveValue('BV');
-    await expect(this.destinationArrival).toHaveValue('asSoonAsPossible');
+    expect(destinationCiValue).toContain(DESTINATION_DETAILS.suggestionResult);
```

Comment:
nit: spread `DESTINATION_DETAILS` keys when you declare it in order to use only the keys, if it takes too much space in the file, import it from a `consts.ts` file:

```js
const {
  destinationInput,
  suggestionResult,
  defaultChValue,
  defaultArrivalType,
  updatedArrivalType,
  defaultArrivalDate,
  defaultArrivalTime,
  defaultTolerance,
  updatedDate,
  updatedHour,
  updatedMinute,
  updatedTimeValue,
  toleranceNegative,
  tolerancePositive,
} = {
  destinationInput: 'South',
  suggestionResult: 'South_station',
  defaultChValue: 'BV',
  defaultArrivalType: 'asSoonAsPossible',
  updatedArrivalType: 'preciseTime',
  defaultArrivalDate: '17/10/24',
  defaultArrivalTime: '00:00',
  defaultTolerance: '-30/+30',
  updatedDate: '18/10/24',
  updatedHour: '01',
  updatedMinute: '35',
  updatedTimeValue: '01:37',
  toleranceNegative: '-75',
  tolerancePositive: '+45',
};
```

or

```js
import { DESTINATION_DETAILS } from "<path>/consts"

const {
  destinationInput,
  suggestionResult,
  defaultChValue,
  defaultArrivalType,
  updatedArrivalType,
  defaultArrivalDate,
  defaultArrivalTime,
  defaultTolerance,
  updatedDate,
  updatedHour,
  updatedMinute,
  updatedTimeValue,
  toleranceNegative,
  tolerancePositive,
} = DESTINATION_DETAILS
```

same for `ORIGIN_DETAILS`